src/ImageSharp/Formats/Jpeg/Common/FastFloatingPointDCT.cs

```diff
@@ -3,47 +3,47 @@
 
 using System.Numerics;
 using System.Runtime.CompilerServices;
-using Block8x8F = SixLabors.ImageSharp.Formats.Jpeg.Common.Block8x8F;
 
 // ReSharper disable InconsistentNaming
-namespace SixLabors.ImageSharp.Formats.Jpeg.GolangPort.Components
+namespace SixLabors.ImageSharp.Formats.Jpeg.Common
 {
     /// <summary>
-    /// Contains forward and inverse DCT implementations
+    /// Contains inaccurate, but fast forward and inverse DCT implementations.
     /// </summary>
-    internal static class DCT
+    internal static class FastFloatingPointDCT
     {
 #pragma warning disable SA1310 // FieldNamesMustNotContainUnderscore
-        private static readonly float C_1_175876 = 1.175876f;
+        private static readonly float C_1_175876 = 1.175875602f;
```

Comment:
wouldn't theses be better as `const` values instead of `readonly static` ones?

Reply:
Wow, worked with this code a lot but haven't spotted this all the time! I think this is because these values were originally defined as `Vector4`.

Reply:
Yeah, we can fix that as we optimize post beta.